### ISSUE TITLE
Allow disabled_rules in editorconfig section

### ISSUE THREAD

**Issue by dan-f** [pinterest/ktlint#672]
I've got a kotlin project in which it's common practice to create files which hold multiple classes. In each of these cases we name the file with a common suffix.

A simple example of this, using the suffix "Things" would be something like:

```kotlin
// FooThings.kt

data class Foo(...)

data class Bar(...)
```

The problem is that we trip the `filename` rule when there's only one such class in one of these files.

Based on this comment - https://github.com/pinterest/ktlint/issues/237#issuecomment-520032996 - I was hopeful I could add an `.editorconfig` rule like so to disable the `filename` error:

```editorconfig
[**/*Things.kt]
disabled_rules = filename
```

However, this doesn't work.

**Comment by Tapchicoma** [comment]
What ktlint version are you using?

Could try to modify your `.editorconfig` to:
```editorconfig
[**/FooThings.kt]
disabled_rules = filename
```
or if it will still fail to:
```editorconfig
[*.{kt,kts}]
disabled_rules = filename
```
and report back if this will work.

**Comment by dan-f** [comment]
Your suggestion "worked" in the sense that it disabled the `filename` rule for every kotlin file. However, the feature I'm looking for is the ability to disable the `filename` rule for kotlin files that end with a specific suffix (other than simply `.kt`).

**Comment by Tapchicoma** [comment]
Interesting, probably there is a bug in `.editorconfig` globs parsing. I need to check it.

**Comment by romtsn** [comment]
Closing in favour of #742, please track it there.
